snap/hooks/configure

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh -e
+
+CONFIG_PATH="$(snapctl get configuration-path)"
+
+# use fallback configuration mechanism if the configuration-path is set
+if [ -n "${CONFIG_PATH}" ]; then
+    # but always give precedence to the configuration-read plug
+    if snapctl is-connected configuration-read; then
+      logger -t ${SNAP_NAME} "Plug 'configuration-read' is connected"
+      exit 0
+    fi
+
+    case "${CONFIG_PATH}" in
+        /root*) ;;
+        *)
+            >&2 echo "Invalid path. The path must be relative to root (starting with '/root')."
+            logger -t ${SNAP_NAME} "Invalid path. The path must be relative to root (starting with '/root')."
+            exit 1
+            ;;
+    esac
+
+    if [ ! -d "${CONFIG_PATH}" ]; then
+        >&2 echo "Error: configuration-path '${CONFIG_PATH}' does not exist."
+        logger -t ${SNAP_NAME} "Error: configuration-path '${CONFIG_PATH}' does not exist."
+        exit 1
+    fi
```

Comment:
I don't think you need the first case here.
Simply check that `/root/${CONFIG_PATH}` exists and if not, mention that it must be relative to `/root`